aws-cloudformation-stackset/aws-cloudformation-stackset.json

```
@@ -0,0 +1,309 @@
+            "type": "array",
+            "uniqueItems": true,
+            "insertionOrder": false,
+            "items": {
+                "$ref": "#/definitions/Parameter"
+            }
+        },
+        "PermissionModel": {
+            "description": "Describes how the IAM roles required for stack set operations are created. By default, SELF-MANAGED is specified.",
+            "type": "string",
+            "enum": [
+                "SERVICE_MANAGED",
+                "SELF_MANAGED"
+            ]
+        },
+        "Tags": {
+            "description": "The key-value pairs to associate with this stack set and the stacks created from it. AWS CloudFormation also propagates these tags to supported resources that are created in the stacks. A maximum number of 50 tags can be specified.",
+            "type": "array",
+            "uniqueItems": true,
+            "insertionOrder": false,
+            "maxItems": 50,
+            "items": {
+                "$ref": "#/definitions/Tag"
+            }
+        },
+        "TemplateBody": {
+            "description": "The structure that contains the template body, with a minimum length of 1 byte and a maximum length of 51,200 bytes.",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 51200
+        },
+        "TemplateURL": {
+            "description": "Location of file containing the template body. The URL must point to a template (max size: 460,800 bytes) that is located in an Amazon S3 bucket.",
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 1024
+        }
+    },
+    "required": [
+        "PermissionModel"
+    ],
+    "additionalProperties": false,
+    "createOnlyProperties": [
+        "/properties/PermissionModel"
+    ],
+    "writeOnlyProperties": [
+        "/properties/TemplateURL",
+        "/properties/OperationPreferences"
+    ],
+    "readOnlyProperties": [
+        "/properties/StackSetId"
+    ],
+    "primaryIdentifier": [
+        "/properties/StackSetId"
+    ],
+    "handlers": {
+        "create": {
+            "permissions": [
+                "s3:GetObject",
+                "cloudformation:CreateStackSet",
+                "cloudformation:CreateStackInstances",
+                "cloudformation:DescribeStackSetOperation",
+                "cloudformation:TagResource"
+            ]
+        },
+        "read": {
+            "permissions": [
+                "cloudformation:DescribeStackSet",
+                "cloudformation:ListStackInstances",
+                "cloudformation:DescribeStackInstance"
+            ]
+        },
+        "update": {
+            "permissions": [
+                "s3:GetObject",
+                "cloudformation:UpdateStackSet",
+                "cloudformation:CreateStackInstances",
+                "cloudformation:DeleteStackInstances",
+                "cloudformation:UpdatetackInstances",
```

```suggestion
                "cloudformation:UpdateStackInstances",
```

https://docs.aws.amazon.com/IAM/latest/UserGuide/list_awscloudformation.html#awscloudformation-actions-as-permissions

---

Nice catch. Will fix it.